README.md

```diff
@@ -42,3 +42,312 @@ Run linters.
 cargo make clippy
 ```
 
+## Usage
+
+In this section is presented the integration of this library into your application, enabling you to kickstart 
+CQRS/Event sourcing implementation.
+
+### Event sourcing
+
+> Event sourcing is a software architectural pattern used to capture and store the changes or events that occur in an 
+application's state over time. Instead of persisting the current state of an object or entity, event sourcing focuses 
+on recording a sequence of events that have led to that state.
+
+#### Aggregate
+
+To initiate the implementation of CQRS/Event sourcing in your codebase, the initial building block to create is an `Aggregate`.
```

Comment:
```suggestion
To start the implementation of CQRS/Event sourcing in your codebase, the initial building block you need to create is an `Aggregate`.
```

Reply:
I think
```suggestion
To implement CQRS/Event sourcing in your codebase, you need to start by creating an `Aggregate`.
```
is a bit better than what Cristiano suggested?
We could also do
```suggestion
To start implementing of CQRS/Event sourcing in your codebase, first you need to create an `Aggregate`.
```

Reply:
I think that i will go impersonal to maintain the overall style being that above we agreed on changing text to `This section presents the integration of the `

Reply:
I think
> To start implementing of CQRS/Event sourcing in your codebase, first you need to create an `Aggregate`.
is impersonal, but flows a bit better